Tests/test_PDBList.py

```diff
@@ -24,21 +25,27 @@ class TestPBDListGetList(unittest.TestCase):
 
     def test_get_recent_changes(self):
         """Tests the Bio.PDB.PDBList.get_recent_changes method."""
-        pdblist = PDBList(obsolete_pdb="unimportant")  # obsolete_pdb declared to prevent from creating the "obsolete" directory
+        pdblist = PDBList(
+            obsolete_pdb="unimportant"
+        )  # obsolete_pdb declared to prevent from creating the "obsolete" directory
```

Comment:
Here (and the next few cases), I would put the comment on the line above - then the code does not need to split over three lines.